category.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/Style.css">
+    <title>Category</title>
+</head>
+<body id="change-background">
+    <header></header>
+    <main>
+<table>
+
+</table>
+
+    </main>
+    <footer></footer>
+    <!-- <script type="text/javascript" src="./js/Quiz.json"></script> -->
```

Comment:
do not push commented lines

Reply:
I also thought I deleted all comments ( X